README.md

```diff
@@ -1,3 +1,13 @@
+# Closure Rules for Bazel for Yext internal use
+
+To build/test locally you must pass the --proto_compiler flag whihc we 
+use to speed up builds.
+
+```
+bazel build //... --proto_compiler=@com_google_protobuf//:protoc
```

Comment:
Heads up, if we confirm the `.bazelrc` was enough, we don't have to pass in these flags here

Reply:
true, i can update. It does work properly with the bazel rc change